pkg/liqo-controller-manager/storageprovisioner/localprovisioner.go

```diff
@@ -134,3 +135,11 @@ func mergeAffinities(vol1, vol2 *v1.PersistentVolumeSpec) *v1.VolumeNodeAffinity
 		Required: &selector,
 	}
 }
+
+func getLocalRealPvcName(virtualPvcNamespace, virtualPvcName string) string {
+	pvcName := fmt.Sprintf("%s-%s", virtualPvcNamespace, virtualPvcName)
+	if len(pvcName) > 63 {
+		return pvcName[:63]
+	}
+	return pvcName
+}
```

Comment:
This could nonetheless lead to collisions in case of long namespace and PVC names.
To avoid this, I would personally keep the end of the name (instead of the beginning), to prevent problems with the PVCs originated from statefulsets.
Wondering also whether adding an hash or something similar, instead of the full namespace name, could lower the risk...

Reply:
I used the parent PVC UID, in that way the real PVC name is the same as the virtual PV, and simpler to understand the relation between them (compared to the hash)

Reply:
Makes sense to me. Could it be worth adding a couple of labels identifying the virtual PVC namespace and name?